packages/purpose-process/src/utilities/config.ts

```diff
@@ -14,10 +14,10 @@ const PurposeProcessConfig = CommonHTTPServiceConfig.and(ReadModelDbConfig)
   .and(
     z
       .object({
-        RISK_ANALYSIS_DOCUMENTS_PATH: z.string(),
+        RISK_ANALYSIS_PATH: z.string(),
```

Comment:
since this rename requires an update in the deployment configuration, if it is not required may I ask you to revert it? 🙏

Reply:
Sure! :)